src/docs/markdown/build.md

```diff
@@ -4,10 +4,24 @@ title: "Build from source"
 
 # Build from source
 
+There are 2 ways to build Caddy, as opposed to installation of pre-built executable:
```

Comment:
> as opposed to installation of pre-built executable

I think we should change this to something like "if you need to customize Caddy" or something like that. "as opposed to" sounds negative